hnn_core/parallel_backends.py

```diff
@@ -12,10 +12,36 @@
 from warnings import warn
 from subprocess import Popen
 import selectors
+import binascii
 
 _BACKEND = None
 
 
+def _clone_and_simulate(net, trial_idx, prng_seedcore_initial):
+    """Run a simulation including building the network
+
+    This is used by both backends. MPIBackend calls this in mpi_child.py, once
+    for each trial (blocking), and JoblibBackend calls this for each trial
+    (non-blocking)
+    """
+
+    # avoid relative lookups after being forked (Joblib)
+    from hnn_core.network_builder import NetworkBuilder
+    from hnn_core.network_builder import _simulate_single_trial
+
+    # XXX this should be built into NetworkBuilder
+    # update prng_seedcore params to provide jitter between trials
+    for param_key in net.params['prng_*'].keys():
+        net.params[param_key] += trial_idx
```

Comment:
```suggestion
    for param_key in prng_seedcore_initial.keys():
        net.params[param_key] = prng_seedcore_inital[param_key] + trial_idx
```

Reply:
Since the test I added in #171 only runs 2 trials when validating consistent jitter across `MPIBackend` and `JoblibBackend`, this issue wasn't caught. In a nutshell, both backends need to iterate their prng_seedcore values like 1+0=1, 1+1=2, 1+2=3, 1+3=4, etc.; however, I suspect `MPIBackend` will iterate like 1+0=1, 1+1=2, 2+2=4, 4+3=7, etc.

Reply:
Thanks! It looks like a copy and paste error, where `prng_seedcore_initial` wasn't even used in the code. WIll change.